pano.py

```diff
@@ -33,7 +33,7 @@
     from pygame.locals import *
     from pygame import camera
 except ImportError:
-    print 'Error in import Pygame. This activity requires Pygame 1.9'
+    print ('Error in import Pygame. This activity requires Pygame 1.9')
```

Comment:
Does not require 1.9, the message should either omit the version, or just don't handle the ImportError.

Your edit also triggered flake8 E211.

Reply:
What can be done here I am unsure?

Reply:
Unfortunately you have gone against our contributor's guide recommendation not to use your _master_ branch, otherwise I could just fix it and push a commit to the pull request in order to show you.

```patch
--- a/pano.py
+++ b/pano.py
@@ -28,12 +28,9 @@ from gi.repository import Gtk
 import time
 import logging
 import stitcher
-try:
-    import pygame
-    from pygame.locals import *
-    from pygame import camera
-except ImportError:
-    print ('Error in import Pygame. This activity requires Pygame 1.9')
+import pygame
+from pygame.locals import *
+from pygame import camera
 
 
 class PanoCapture():
```

Reply:
The wildcard import is not your problem for this pull request, but it is something that can be fixed later.  It makes flake8 errors and inhibits some of flake8 feature set.